Plugins/Bugsnag/Source/Bugsnag/Private/Apple/Specs/ApplePlatformConfiguration.spec.cpp

```diff
@@ -180,11 +180,22 @@ void FApplePlatformConfigurationSpec::Define()
 			It("Telemetry", [this]()
 				{
 					TSharedRef<FBugsnagConfiguration> Configuration = MakeShared<FBugsnagConfiguration>(ApiKey);
+
 					BugsnagConfiguration* CocoaConfig = FApplePlatformConfiguration::Configuration(Configuration);
-					TEST_TRUE(CocoaConfig.telemetry == BSGTelemetryInternalErrors);
+
+					TEST_TRUE(CocoaConfig.telemetry == BSGTelemetryAll);
+
 					Configuration->SetTelemetry(EBugsnagTelemetryTypes::None);
+
 					CocoaConfig = FApplePlatformConfiguration::Configuration(Configuration);
+
 					TEST_TRUE(CocoaConfig.telemetry == 0);
+
+					Configuration->SetTelemetry(EBugsnagTelemetryTypes::Usage);
+
+					CocoaConfig = FApplePlatformConfiguration::Configuration(Configuration);
+
+					TEST_TRUE(CocoaConfig.telemetry == BSGTelemetryUsage);
```

Comment:
Did you mean to leave all these blank lines in? Looks out of place next to the other test cases.

Reply:
fixed